includes/admin/class-wc-stripe-rest-payment-intents-controller.php

```
@@ -0,0 +1,244 @@
+		if ( is_wp_error( $response ) ) {
+			return $response;
+		}
+
+		if ( is_object( $response ) && isset( $response->error ) ) {
+			$error_code    = isset( $response->error->code ) ? (string) $response->error->code : 'wc_stripe_api_error';
+			$error_message = isset( $response->error->message ) ? (string) $response->error->message : __( 'Stripe API returned an error.', 'woocommerce-gateway-stripe' );
+
+			return new WP_Error( $error_code, $error_message, [ 'status' => 400 ] );
+		}
+
+		$filtered_response = WC_Stripe_REST_Response_Filter::filter_response( $response, $this->stripe_response_allowed_fields );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_REST_Response_Filter' in method 'get_payment_intents'. (undefined)

(StaticAccess)